npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/vulnerability_auditor.rb

```diff
@@ -90,6 +90,13 @@ def viable_audit_result?(audit_result, security_advisories)
           validation_result = validate_audit_result(audit_result, security_advisories)
           return true if validation_result == :viable
 
+          case validation_result
```

Comment:
Could this be placed after the logging on line 100 so that we can get/share the insight? As it is, I'd expect execution to stop with `raise`